src/main.css

```diff
@@ -23,4 +23,34 @@
       0 20%
     );
   }
+
+  .perspective {
+    perspective: 350px;
+  }
+
+  .fade-text::before,
+  .fade-text::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    right: 0;
+    pointer-events: none;
+    z-index: 1;
+  }
+
+  .fade-text::before {
+    top: 0;
+    height: 350px;
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0));
+  }
+
+  .fade-text::after {
+    bottom: 0;
+    height: 10px;
+    background: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0));
+  }
```

Comment:
Tout ça peut aller dans les class tailwind
[doc pseudo elements](https://tailwindcss.com/docs/hover-focus-and-other-states#pseudo-elements)